plugins/woocommerce/includes/tracks/events/class-wc-settings-tracking.php

```diff
@@ -59,10 +59,16 @@ class WC_Settings_Tracking {
 	public function init() {
 		add_action( 'woocommerce_settings_page_init', array( $this, 'track_settings_page_view' ) );
 		add_action( 'woocommerce_update_option', array( $this, 'add_option_to_list' ) );
+		add_action( 'woocommerce_update_shipping_setting', array( $this, 'add_option_to_list_and_track_change' ) );
 		add_action( 'woocommerce_update_options', array( $this, 'send_settings_change_event' ) );
 		add_action( 'admin_enqueue_scripts', array( $this, 'possibly_add_settings_tracking_scripts' ) );
 	}
 
+	public function add_option_to_list_and_track_change( $option ) {
```

Comment:
This function is not tracking a change. It is marking an option as updated.

So, what about `add_option_to_list_and_mark_as_updated`?